custom_components/elegoo_printer/definitions.py

```
@@ -191,3 +191,13 @@ class ElegooPrinterSensorEntityDescription(
         value_fn=lambda self: self.coordinator.data.status.temp_target_box,
     ),
 )
+
+PRINTER_IMAGES: tuple[ElegooPrinterSensorEntityDescription, ...] = (
+    ElegooPrinterSensorEntityDescription(
+        key="cover_image",
+        translation_key="cover_image",
+        value_fn=lambda _printer: _printer.get_current_print_thumbnail(),
+        available_fn=lambda _printer: _printer.get_current_print_thumbnail()
+        is not None,
+    ),
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'key' in constructor call

(E1123)